README.md

```diff
@@ -1,6 +1,6 @@
 # Onfido OpenAPI specification (beta)
 
-:warning: Our OpenAPI specification is currently in beta and we welcome any feedback. You can contact us via the issues tab or [email](mailto:openapi-feedback@onfido.com), but we don't yet officially support this specification. :warning:
+:warning: Our OpenAPI specification is currently in beta and we welcome any feedback. You can contact us via the issues tab or [support portal](https://public.support.onfido.com), but we don't yet officially support this specification. :warning:
```

Comment:
Should we consider that the spec is out of beta since we have all the reports ?

Reply:
I think we should wait a little bit longer, to have one client library sufficiently tested.